packages/config/config/devices/0x010f/fgms001.json

```diff
@@ -466,7 +508,7 @@
 			"#": "40",
 			"$if": "firmwareVersion >= 3.2",
 			"label": "Illuminance report - threshold",
-			"description": "Change in light intensity level resulting in illumination report",
+			"description": "Minimum change in light intensity level resulting in illumination report (lux).",
```

Comment:
The unit should be specified using the `"unit"` property, not as part of the description.